docs/sources/administration/team-management/configure-grafana-teams.md

```diff
@@ -0,0 +1,213 @@
+1. Click the name of the Team to which you want to add members, and click **+ Add member**.
+1. Search for and select a user.
+1. Choose whether to add the user as a Team **Member** or **Admin**.
+1. Click **Save**.
+
+## Grant or change Team member permissions
+
+Complete this task when you want to add or modify team member permissions.
+
+To grant team member permissions:
+
+1. Sign in to Grafana as an `org administrator` or `team administrator`.
+1. Click the arrow next to **Administration** in the left-side menu, click **Users and access**, and select **Teams**. 
+1. Click the pencil next to the name of the Team for which you want to add or modify team member permissions.
+1. In the team member list, locate the user that you want to change. You can use the search field to filter the list if necessary.
+1. Under the **Permission** column, select the new permission level. 
+
+## Add roles to a Grafana Team
+
+You can add or delete roles from a specified team.  
+
+To add a role, complete the following steps:
+
+1. Sign in to Grafana as an `org administrator` or `team administrator`.
+1. Click the arrow next to **Administration** in the left-side menu, click **Users and access**, and select **Teams**. 
+1. Select the Team and click under the **Role** column. Select from a list of current fixed or plugin roles or clear all roles and start over. As you hover over each role a list of permissions appears to the right. You can uncheck any permission for additional fine-grained control.
+1. Click **Update** to add the new role or roles. 
+
+To delete a role, remove the check next to the role name and click **Update**. 
+
+## Delete a team
+
+Delete a team when you no longer need it. This action permanently deletes the team and removes all team permissions from dashboards and folders. This task requires that you have `organization administrator` permissions.
+
+1. Sign in to Grafana as an `org administrator` or `team administrator`.
+1. Click the arrow next to **Administration** in the left-side menu, click **Users and access**, and select **Teams**. 
+1. Click the **red X** on the right side of the name of the team.
+1. Click **Delete**.
+
+## Create isolated or collaborative teams
+
+Grafana Teams can either be `isolated` or `collaborative`. Isolated teams can only see their own resources. They cannot see other team’s dashboards, data, alerts, etc. Collaborative teams have access to other team’s resources. Grafana Cloud users must contact Support.
+
+To create an isolated team add the following to the Grafana configuration file:
+
+```
+auto_assign_org_role = None
+role_attribute_path = contains(groups[*], 'admin') && 'Admin' || 'None'
+```
+
+To create a collaborative team add the following to the Grafana configuration file:
+
+```
+auto_assign_org_role = Viewer
+role_attribute_path = contains(groups[*], 'admin') && 'Admin' || 'None'
+```
+You can also use a terraform script as shown in the following example:
+
+```terraform
```

Comment:
For syntax highlighting, you actual want `hcl` which stands for Hashicorp Configuration Language.

````suggestion
```hcl
````